cibuildwheel/macos.py

```diff
@@ -89,6 +90,7 @@ def call(args, env=None, cwd=None, shell=False):
 
         # install pip & wheel
         call(['python', get_pip_script, '--no-setuptools', '--no-wheel'], env=env)
+        assert os.path.exists(os.path.join(installation_bin_path, 'pip'))
         call(['pip', '--version'], env=env)
         call(['pip', 'install', '--upgrade', 'setuptools'], env=env)
         call(['pip', 'install', 'wheel'], env=env)
```

Comment:
It feels weird no to upgrade `wheel` (and thus not having a single line: `call(['pip', 'install', '--upgrade', 'setuptools', 'wheel'], env=env))`